plugins/connection/vmware_tools.py

```diff
@@ -165,7 +165,7 @@
             - name: ansible_vmware_tools_executable
 '''
 
-EXAMPLES = r'''
+example = r'''
```

Comment:
`yamllint` is not liking this `EXAMPLES` for some reason. Renamed this variable for removing following error. Also, None of the connection plugin provide `EXAMPLES`.

```
ERROR: plugins/connection/vmware_tools.py:23:1: unparsable-with-libyaml: expected a single document in the stream - but found another document
```

Reply:
Can you try to put the `----`  at the top of the document?